SU-HSDO/suhumsci
# Shs 125 some accessibility improvements

## Summary
This addresses some of the accessibility violations for SHS-125. The ticket has a lot of bullet points that were mainly split out into their own tickets.

This PR fixes two items:
* It revamps the original menu to use buttons instead of links when there is no link
* It adds focus outline to form elements

## Steps to Test
1. Visit a site with the original menu. Verify it still works across all screen sizes. Verify that visually nothing has changed. Also check that the navigation works with keyboards and with VoiceOver
2. Tab through an element with a form field (the search works) and verify that it now has an outline

## Screenshots
![Screen Shot 2022-12-06 at 8 52 00 AM](https://user-images.githubusercontent.com/8405274/205930341-b07a7421-052f-437b-9f62-4e57c9cac4ee.png)

![Screen Shot 2022-12-06 at 8 52 18 AM](https://user-images.githubusercontent.com/8405274/205930369-dc0c9454-2962-4d63-a266-932c77abecaf.png)

## Notes
We'll probably want to include this in the next release and not the one going out tomorrow.
